sopel/config/core_section.py

```diff
@@ -336,6 +340,22 @@ def homedir(self):
     capabilities.
     """
 
+    commands_on_connect = ListAttribute('commands_on_connect')
+    r"""A list of commands to perform upon successful connection to IRC server.
+
+    When entered using the config wizard, commas will be escaped automatically.
+    Otherwise, commas must be escaped, e.g.: ``PRIVMSG Q@CServe.quakenet.org
+    :AUTH my_username MyPassword\,HasAComma@#$%!`` Nothing else needs to be
```

Comment:
This, too, needs updating now that escaped commas aren't a thing any more.

Eliminating that false escape sequence means the whole docstring can quit being raw (`r"""`) also.